x-pack/filebeat/module/azure/activitylogs/test/activity_log_expected.json

```diff
@@ -0,0 +1,103 @@
+            "id" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+            "type" : "filebeat",
+            "ephemeral_id" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+            "version" : "8.0.0"
+        },
+        "ecs" : {
+            "version" : "1.1.0"
+        },
+        "identity" : {
+            "authorization" : {
+                "evidence" : {
+                    "roleAssignmentScope" : "/subscriptions/2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+                    "roleDefinitionId" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+                    "role" : "Azure EventGrid Service BuiltIn Role",
+                    "roleAssignmentId" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+                    "principalId" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+                    "principalType" : "ServicePrincipal"
+                },
+                "scope" : "/subscriptions/2a7e2503-d7e2-405a-a84c-c333b9f7cb73/resourceGroups/sa-hemant/providers/Microsoft.EventHub/namespaces/azurelsevents/authorizationRules/RootManageSharedAccessKey",
+                "action" : "Microsoft.EventHub/namespaces/authorizationRules/listKeys/action"
+            },
+            "claims" : {
+                "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/nameidentifier" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+                "ver" : "1.0",
+                "http://schemas.microsoft.com/identity/claims/identityprovider" : "https://sts.windows.net/2a7e2503-d7e2-405a-a84c-c333b9f7cb73/",
+                "aio" : "2a7e2503-d7e2-40",
+                "iss" : "https://sts.windows.net/2a7e2503-d7e2-405a-a84c-c333b9f7cb73/",
+                "uti" : "2a7e2503-d7e2",
+                "aud" : "https://management.core.windows.net/",
+                "nbf" : "1569500393",
+                "appidacr" : "2",
+                "http://schemas.microsoft.com/identity/claims/tenantid" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+                "appid" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+                "http://schemas.microsoft.com/identity/claims/objectidentifier" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+                "exp" : "1569529493",
+                "iat" : "1569500393"
+            }
+        },
+        "host" : {
+            "hostname" : "DESKTOP-RFOOE09",
+            "os" : {
+                "build" : "17134.1006",
+                "kernel" : "10.0.17134.1006 (WinBuild.160101.0800)",
+                "name" : "Windows 10 Pro",
+                "family" : "windows",
+                "version" : "10.0",
+                "platform" : "windows"
+            },
+            "name" : "DESKTOP-RFOOE09",
+            "id" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+            "architecture" : "x86_64"
+        },
+        "correlationId" : "2a7e2503-d7e2-405a-a84c-c333b9f7cb73",
+        "event" : {
+            "module" : "azure",
+            "dataset" : "azure.activitylogs"
+        },
+        "resultType" : "Failure",
```

Comment:
I think some of these fields could be mapped into ECS like

- `resultType` -> `event.result`
- `callerIpAddress` -> `source.ip`

I'm sure they're are more and maybe you are already planning for some of these mappings.

Reply:
@andrewkroh , thanks, in the course of identifying the ecs fields. I could not find event.result in https://www.elastic.co/guide/en/beats/filebeat/master/exported-fields-ecs.html. Is it event.outcome or I am not looking in the right list.

Reply:
You're right. It's `event.outcome`. I was working from memory on that example.